Products/Zuul/routers/monitor.py

```diff
@@ -37,7 +37,7 @@ def getTree(self, id):
         @return:  Object representing the tree
         """
         facade = self._getFacade()
-        monitors = facade.queryPerformanceMonitors()
+        monitors = facade.queryMonitors()
```

Comment:
I don't see query monitors in the facade either https://github.com/zenoss/zenoss-prodbin/blob/develop/Products/Zuul/facades/monitor.py